pkgs/top-level/all-packages.nix

```diff
@@ -6836,6 +6836,8 @@ with pkgs;
 
   dmd = callPackage ../development/compilers/dmd {
     inherit (darwin.apple_sdk.frameworks) Foundation;
+    # https://github.com/NixOS/nixpkgs/pull/206907#issuecomment-1527034123
+    stdenv = gcc11Stdenv;
```

Comment:
```suggestion
    stdenv = if stdenv.isLinux then gcc11Stdenv else stdenv;
```

Reply:
For some reason github is not letting me apply your suggestion ("This diff has recently been updated.  Refresh and try again" -- but I refresh and still get the error).

I will apply your suggestion manually.